src/client/fuse_ll.cc

```diff
@@ -193,7 +193,7 @@ static void fuse_ll_lookup(fuse_req_t req, fuse_ino_t parent, const char *name)
 }
 
 static void fuse_ll_forget(fuse_req_t req, fuse_ino_t ino,
-			   long unsigned nlookup)
+			   uint64_t nlookup)
```

Comment:
@vbashkirtsev could you instead have something like

```c++
#if FUSE_VERSION >= FUSE_MAKE_VERSION(3, 0)
using nlookup_t = uint64_t;
#else
using nlookup_t = long unsigned;
#endif
static void fuse_ll_forget(fuse_req_t req, fuse_ino_t ino,
			   nlookup_t nlookup)
```

Reply:
Yeah... Always forgetting about conditional compilation... :( Amended the commit but decided to go without nlookup_t and used complete original fuse signatures for clarity.